.github/workflows/qctools.yml

```diff
@@ -217,6 +196,22 @@ jobs:
         echo "::add-path::C:/msys64/usr/bin"
         echo "::add-path::C:/msys64/mingw64/bin"
 
+    - name: Install FFmpeg
+      run: |
+        cd ..
+        $FFMPEG_LATEST_WIN64 = "ffmpeg-4.2.1-win64"
```

Comment:
Nit: the latest FFmpeg release for Windows is currently 4.2.2.

Reply:
Changed to `FFmpeg` master branch, thanks a lot for spotting that! :)

Reply:
Thx @Luni-4, we are using some libavfilter options that are after 4.2.2 so need to use master at this time.

Reply:
I see, thanks for the explanation @dericed :)